migrationConsole/build.gradle

```diff
@@ -27,10 +27,13 @@ def workflowTemplatesStagingP =
 tasks.register('installDependencies', NpmTask) {
     description = 'Install npm dependencies for orchestrationSpecs'
     workingDir = orchestrationSpecsDirP.asFile
-    args = ['ci', '--force', '--omit=optional']
+    args = ['ci', '--force']
```

Comment:
Why did you need to do this?  I had a heck of a time getting the npm packages to build the first time around because of native code - specifically, tsgo.  The --omit=optional fixed those issues.